### PR TITLE
56 assign shootcommand and backoffcommand to custom trigger mapped to lt axis

### LINKED ISSUE #56: Assign ShootCommand and BackOffCommand to custom trigger mapped to 'RT' axis
- [x] Create custom joystick axis trigger with constructor that accepts a threshold value
- [x] Simplify FireCommand by removing its DoubleSupplier parameter
- [x] Map single button shooting to LT, using a small threshold
  - [x] .whenPressed is a ReverseHopperCommand with small timeout
  - [x] .whileHeld is a ParallelCommandGroup
    - [x] RevCommand
    - [x] FireCommand after short delay

### PR DESCRIPTION
**Pull request by fishyfishfish55**
Creates "one trigger shooting", using a custom AnalogButton based off an analog axis.

### CODE REVIEW

File: src/main/java/org/frc571/bradley/Constants.java [hunk 1]
```diff
@@ -37,8 +37,8 @@
         public static final double MAX_AXIS = 1.0;
         public static final double kDeadzone = 0.3;
         public static final double kShootMotor = 0.5;
-        public static final double FIRE_COMMAND_END = 0.1;
-        public static final double FIRE_COMMAND_THRESHOLD = 0.5;
+        public static final double FIRE_COMMAND_THRESHOLD = 0.3;
+        public static final double REV_COMMAND_THRESHOLD = 0.1;
```
Comment (kramoniker) [inline src/main/java/org/frc571/bradley/Constants.java]: This is no longer required.

File: src/main/java/org/frc571/bradley/Constants.java [hunk 2]
```diff
@@ -55,7 +55,12 @@
         public static final double kIntakeMotorSpeed = 0.50;
         public static final double RAISE_INTAKE_SPEED = 0.3;
         public static final double FLYWHEEL_MAX_SPEED = 0.79;
-        public static final double RAMP_TIME = 0.79;
+        public static final double RAMP_TIME = 0.4;
```
Comment (kramoniker) [inline src/main/java/org/frc571/bradley/Constants.java]: This should be in TimeConstants

File: src/main/java/org/frc571/bradley/RobotContainer.java.orig [hunk 3]
```diff
@@ -0,0 +1,169 @@
+package org.frc571.bradley;
```
Comment (kramoniker) [inline src/main/java/org/frc571/bradley/RobotContainer.java.orig]: Should not have added this file.

**Review by kramoniker — CHANGES_REQUESTED** [review]

File: src/main/java/org/frc571/bradley/Constants.java (hunk 1 above)
Comment (fishyfishfish55) [reply src/main/java/org/frc571/bradley/Constants.java]: Removed FIRE_COMMAND_THRESHOLD; REV_COMMAND_THRESHOLD is needed in RobotContainer. Renamed to SHOOT_TRIGGER_THRESHOLD,

File: src/main/java/org/frc571/bradley/Constants.java (hunk 2 above)
Comment (fishyfishfish55) [reply src/main/java/org/frc571/bradley/Constants.java]: Moved to TimeConstants

File: src/main/java/org/frc571/bradley/RobotContainer.java.orig (hunk 3 above)
Comment (fishyfishfish55) [reply src/main/java/org/frc571/bradley/RobotContainer.java.orig]: Deleted this file

**Review by kramoniker — APPROVED** [review]

**Pull request merged**